shell/src/mempool/mempool_state.rs

```diff
@@ -224,6 +243,18 @@ impl MempoolState {
     pub fn operations(&self) -> &HashMap<OperationHash, Operation> {
         &self.operations
     }
+
+    // pub fn add_stream(&mut self, id: Uuid, waker: Waker) {
```

Comment:
@adonagy 
do we need this?
and please, check easy clippy stuff with `&` in this file